### ISSUE TITLE
The Legend of Zelda: TOTK RDNA3-exclusive Vulkan Issues

### ISSUE THREAD

**Issue by SamyolGH** [yuzu-emu/yuzu#10228]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Early Access 3578

### Description of Issue

![git1](https://github.com/yuzu-emu/yuzu/assets/88501468/7a988941-f00d-4e70-96ee-7c3cdde5124a)
![git2](https://github.com/yuzu-emu/yuzu/assets/88501468/c509cbe1-d675-40b2-ac38-972b21956af7)

### Expected Behavior

Game operates without artifacts appearing on the ground as shown in the images.

### Reproduction Steps

Launch Tears of the Kingdom with a 7900XT/X with Vulkan enabled watch the issues ensue.

### Log File

(N/A)

### System Configuration

CPU: AMD Ryzen 9 7950X
GPU/Driver: Radeon RX 7900XTX (23.4.3)
RAM: 64GBs DDR5 @ 6000MHz
OS: Windows 11 (22621.1702)

**Comment by FernandoS27** [comment]
this looks similar to a BotW bug with anisotropic filtering. have you tried setting it to Default?

**Comment by Moonlacer** [comment]
If you can, answer the above questions, and if we can determine they are RDNA3, I will report them to AMD. There are some issues that occur in both games that 100% are RDNA3 specific

**Comment by SamyolGH** [comment]
> this looks similar to a BotW bug with anisotropic filtering. have you tried setting it to Default?

Yes, have tried all levels of anisotropic filtering. Was curious if there was a way at all to build Yuzu with it entirely disabled.

> Do these issues appear with mesa radv?

Nope! Issue is not present on mesa RADV.

> If you can, answer the above questions, and if we can determine they are RDNA3, I will report them to AMD. There are some issues that occur in both games that 100% are RDNA3 specific

I cannot 100% confirm this is RDNA3 as I do not have access to any other GPUs. However, I have not been made aware of anyone making similar reports on NVIDIA or pre-RDNA3 AMD cards nor Intel GPUs.

**Comment by liamwhite** [comment]
To remove anisotropy at build time, you can change this to `0.0f`:
```
src/video_core/renderer_vulkan/vk_texture_cache.cpp:        .maxAnisotropy = max_anisotropy,
```

**Comment by SamyolGH** [comment]
Removed antisotropy at build time and the issue persist.

**Comment by Moonlacer** [comment]
We have started the process of working with AMD on a fix, thank you for your patience

**Comment by Alias8686** [comment]
I know your working on this but I Just thought I'd add some more screenshots, as I've only just upgraded from a 6800XT to a 7900XTX today and had no visual issues here with the previous card.

![0100f2c0115b6000_2023-05-12_15-11-08-605](https://github.com/yuzu-emu/yuzu/assets/133318383/f1cc50dc-bba0-4748-816e-106d4111873b)
![0100f2c0115b6000_2023-05-12_15-11-18-242](https://github.com/yuzu-emu/yuzu/assets/133318383/e7241a23-409b-4a70-83c8-f955642af2a6)
![0100f2c0115b6000_2023-05-12_15-17-28-490](https://github.com/yuzu-emu/yuzu/assets/133318383/4e861e5f-00b6-474e-8519-3a26692d329a)

**Comment by VicBiper** [comment]
Hi,

Same isuue here. I have the XFX 7900 XTX with the exact same problem.

**Comment by MikeHuntington** [comment]
Just wanted to confirm that I'm also seeing this with 7900 XT driver version 23.3.1

**Comment by henres** [comment]
Hello,
Wanted to confirm this bug too on linux with 7900 XTX.

![Snapshot_2023-05-12_18-09-15](https://github.com/yuzu-emu/yuzu/assets/866305/fb957045-99b4-40b0-9e3b-27ce46842d23)

**Comment by SamyolGH** [comment]
> Hello,
> Wanted to confirm this bug too on linux with 7900 XTX.

On Linux? That's unusual, I did not experience the issue using RADV.

**Comment by nick-davison** [comment]
Also having this issue on Windows 11 with 7900XT.

**Comment by danwpeters** [comment]
7900 XTX - Driver Version 23.4.3 - With the same problem.

**Comment by Moonlacer** [comment]
I would also appreciate someone confirming the artifacting on the latest Meda with stock settings, and preferably a picture along with it

**Comment by Kouseilink** [comment]
> We have started the process of working with AMD on a fix, thank you for your patience

Is there a general timeframe for how long this could take to fix? Thank you for so quickly raising this with amd

**Comment by Moonlacer** [comment]
> > We have started the process of working with AMD on a fix, thank you for your patience
> 
> Is there a general timeframe for how long this could take to fix? Thank you for so quickly raising this with amd

Absolutely no timeframe, but they do know with it being a new release it is fairly important to fix

**Comment by Shidoribp** [comment]
Does AMD need to update they Drivers or is Yuzu fixing this issue?

**Comment by VicBiper** [comment]
> Does AMD need to update they Drivers or is Yuzu fixing this issue?

I think both…

**Comment by SamyolGH** [comment]
> > Does AMD need to update they Drivers or is Yuzu fixing this issue?
> 
> I think both…

No point in dwelling on whether AMD or Yuzu is at fault, just hope for a fix.

**Comment by henres** [comment]
> > Hello,
> > Wanted to confirm this bug too on linux with 7900 XTX.
> 
> On Linux? That's unusual, I did not experience the issue using RADV.

I Will check which driver I use when I can.

**Comment by henres** [comment]
> > > Hello,
> > > I Wanted to confirm this bug too on Linux with 7900 XTX.
> > 
> > 
> > On Linux? That's unusual, I did not experience the issue using RADV.
> 
> I Will check which driver I use when I can.

Hello,
I can confirm that the bug happened with amdvlk (and that proves that it's a driver bug).
It worked well with RADV.
Thanks

**Comment by HoRnEyDvL** [comment]
Same issue with 7900 xtx sapphire on 5950x . Artifacts especially noticeable against cave walls etc.

Switching to OpenGL, and no graphics issues however take a hit with FPS.

Latest AMD driver 23.4.3 and Yuzu EA 3588 

Looking forward to a Vulkan Fix

![image](https://github.com/yuzu-emu/yuzu/assets/73101201/e35fb95c-7920-403c-a211-c7ca52cb632f)
![image](https://github.com/yuzu-emu/yuzu/assets/73101201/6074a3eb-7704-48de-bc29-407c3c6e87b1)
![image](https://github.com/yuzu-emu/yuzu/assets/73101201/e53ddba0-3761-480c-8b0a-d21c258a4314)

**Comment by Ombretoile** [comment]
Same problem as everyone.

![kbnay75j0sza1](https://github.com/yuzu-emu/yuzu/assets/133567272/94e6725a-9e05-447b-884b-f26459adc3b1)

**Comment by itsthebrodie** [comment]
7900 xt, 5800X3D. having the same issue. looking forward to a fix or work around

**Comment by KeennyyW** [comment]
I have the same issue with rx7900xt

![Screenshot 2023-05-14 185325](https://github.com/yuzu-emu/yuzu/assets/101289752/3dda1f39-7e3a-4bea-9a2c-93e4e2b004a0)

**Comment by offbrandd** [comment]
Any news one this? The effect is pretty egregious in many areas

**Comment by Moonlacer** [comment]
> Any news one this? The effect is pretty egregious in many areas

It's in AMD's hands now

**Comment by cmagk** [comment]
Same problem with 7900 XT.
Are there any workarounds to reduce the problem right now?

**Comment by Kouseilink** [comment]
Going 4x resolution minimises this specific issue to the point where I've played about 40 hours and not noticed it that much, I'd say it's 85-90% reduced

**Comment by cmagk** [comment]
> Going 4x resolution minimises this specific issue to the point where I've played about 40 hours and not noticed it that much, I'd say it's 85-90% reduced 

That worked pretty well.
Thank you, hope an actual solution will be released soon

**Comment by Moonlacer** [comment]
Hello all, please keep this issue clear of as many messages as possible. We will let you know of any updates if they are not under our NDA with AMD. We know all RDNA3 users face this issue, and we thank you for all the information you've provided us with

**Comment by itsthebrodie** [comment]
> Going 4x resolution minimises this specific issue to the point where I've played about 40 hours and not noticed it that much, I'd say it's 85-90% reduced

tested it myself quickly, didn't help and also took frame hit from higher resolution. thanks for the suggestion tho
